Assets/Scripts/SmallHazardDestroyerScript.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
```

Comment:
This script should be called SmallHazardCollisionHandler

We need to start being consistent in our script names. Collisions are handled in CollisionHandlers. 

We can add these naming conventions to our best practices doc.